src/routes/(console)/project-[region]-[project]/databases/database-[database]/table-[table]/+page.svelte

```diff
@@ -229,6 +230,8 @@
     <!-- CSVs can be text/plain or text/csv sometimes! -->
     <FilePicker
         {onSelect}
+        showLocalFileBucket={true}
```

Comment:
```suggestion
        showLocalFileBucket
```
boolean props equal to true when no value is passed 👍